### ISSUE TITLE
unable to use absolute file paths

### ISSUE THREAD

**Issue by GitMensch** [tenninebt/vscode-koverage#21]
Project is stored in "/home/project" (so that's the workspace folder), the build is done in "/tmp/project" (and therefore coverage files are to be found there.

The following setup does work with coverage-gutters, but doesn't with koverage

```jsonc
    "coverage-gutters.coverageBaseDir": "/tmp/project",
    "koverage.coverageFilePaths": [
        "/tmp/project"
    ],
```

koverage output says:

~~~
"message": "createFileSystemWatcher(Pattern = /home/project/**/{/tmp/project/**})",
"message": "Could not find a Coverage file!",
~~~

Is the configuration wrong or does Koverage really not support absolute names yet (I didn't get it working with ../ entries either)?

**Comment by tenninebt** [comment]
Sorry, been very busy lately. Yes, I think the current design assumes that the coverage files are in the open workspace. 
I'll have a look by the end of this week to see what's blocking and what can be done.
(Obviously, if you have a fix to suggest, please do, just open a pull-request and I'll review it.)

**Comment by GitMensch** [comment]
> Obviously, if you have a fix to suggest, please do, just open a pull-request and I'll review it

Thank you for the offer but this time I even had not looked at a solution.

> I'll have a look by the end of this week to see what's blocking and what can be done.

That's nice.

**Comment by GitMensch** [comment]
Any chance to inspect that, yet?

**Comment by tenninebt** [comment]
Hey, sorry, been extremely busy.

I'm working on it now, a fix should be here this week. I might build a version for you to test before release it, would you have some slack sometime this week to test?

**Comment by GitMensch** [comment]
I should be able to test that shortly after a test .vsix is made availabe anywhere.

**Comment by tenninebt** [comment]
[vscode-koverage-0.0.13-beta.vsix.zip](https://github.com/tenninebt/vscode-koverage/files/10168009/vscode-koverage-0.0.13-beta.vsix.zip)
Give it a shot, and tell how things work out for you. (This version should be able to handle both absolute paths and glob)

**Comment by GitMensch** [comment]
Test result: the path issues are all gone, there is only no calculation if the config is broken.
**But** the coverage is always shown as 100% without any details, even if the old working paths are used (installing the latest version I get 72% and the expected tree with the details).

**Comment by tenninebt** [comment]
> Test result: the path issues are all gone, there is only no calculation if the config is broken. **But** the coverage is always shown as 100% without any details, even if the old working paths are used (installing the latest version I get 72% and the expected tree with the details).

Can I have a screenshot and a bit more details on what you mean by "config is broken"? (Thanks for your help)
Then, even after you fix the config, there is still no calculation you mean (Like the plugin is in a bad state?)? (A recording of the screen or a more explicit use case will really help) 
You can reach me on Discord as tenninebt#3384 if you have time to discuss further, better than github.

**Comment by GitMensch** [comment]
Not sure how direct messaging work on Discord (you know, "new internet noob"), but we can try.
In the meantime: that's one of the sample projects:

![0.0.12](https://user-images.githubusercontent.com/6699539/206001448-e740728e-2b0e-43e6-b994-c5f0b3b2dbca.png)
![0.0.13.beta](https://user-images.githubusercontent.com/6699539/206001215-b4150672-ab36-4e42-835e-b8c5ac558f73.png)

The difference is that this beta result is available a soon as I have a "correct path setup" (including absolute and glob paths) - but it does not show anything else but a single root node with 100%.

**Comment by GitMensch** [comment]
I can confirm that upcoming 0.13 fixes both this and #22
